crates/common/builder/src/builder_api/header.rs

```diff
@@ -0,0 +1,24 @@
+use alloy_primitives::B256;
+use anyhow::Ok;
+use ream_bls::PubKey;
+
+use crate::{BuilderConfig, builder_bid::SignedBuilderBid};
+
+pub async fn get_builder_header(
+    config: BuilderConfig,
+    parent_hash: B256,
+    pubkey: &PubKey,
```

Comment:
```suggestion
pub async fn get_builder_header(
    config: BuilderConfig,
    parent_hash: B256,
    public_key: &PubKey,
```

we might as well fully spell public to be consistent and for readability, please update all other parts of your PR with this change